cloudigrade/util/aws/helper.py

```diff
@@ -122,6 +123,16 @@ def _verify_policy_action(session, action):
                 Attribute='createVolumePermission',
                 OperationType='add',
             )
+        elif action == 'cloudtrail:DescribeTrails':
+            cloudtrail.describe_trails(DryRun=True)
+        elif action == 'cloudtrail:CreateTrail':
+            cloudtrail.create_trail(DryRun=True)
+        elif action == 'cloudtrail:UpdateTrail':
+            cloudtrail.update_trail(DryRun=True)
+        elif action == 'cloudtrail:PutEventSelectors':
+            cloudtrail.put_event_selectors(DryRun=True)
+        elif action == 'cloudtrail:StartLogging':
+            cloudtrail.start_logging(DryRun=True)
```

Comment:
Most of these seem to miss some required attributes, additionally it doesn't look like `DryRun` is one of the supported options: https://boto3.readthedocs.io/en/latest/reference/services/cloudtrail.html

Reply:
Yep. I will fix that 🤦‍♀️

Reply:
Unfortunately, without a `DryRun` option, we might not be able to actually verify that the policy has the correct permission for us. I'll be curious to see if you're able to find an alternative here.

Reply:
This is unfortunate. Nothing is turning up yet but I am still looking. Kind of bummed about it! 😞